client/src/model/selectionModel/SelectionModel.hx

```diff
@@ -102,6 +102,13 @@ class SelectionModel implements Observer extends Observable
 		this.update(this);
 	}
 	
+	public function isClear() : Bool {
+		if (this.selectedComponents.length == 0 && this.selectedEndpoints.length == 0 && this.selectedLinks.length == 0 && this.selectedPorts.length == 0 ) {
+			return true;
+		}
+		return false;
```

Comment:
Or `return this.selectedComponents.length == 0 && this.selectedEndpoints.length == 0 && this.selectedLinks.length == 0 && this.selectedPorts.length == 0 ;`